pkg/local_workflows/report_analytics_workflow.go

```diff
@@ -41,25 +42,32 @@ func reportAnalyticsEntrypoint(invocationCtx workflow.InvocationContext, inputDa
 	url := fmt.Sprintf("%s/rest/api/orgs/%s/analytics", config.GetString(configuration.API_URL), config.Get(configuration.ORGANIZATION))
 
 	for i, input := range inputData {
-		logger.Println(fmt.Sprintf("%s: processing element %d", reportAnalyticsWorkflowName, i))
+		logger.Printf(fmt.Sprintf("%s: processing element %d", reportAnalyticsWorkflowName, i))
 		documentLoader := gojsonschema.NewBytesLoader(input.GetPayload().([]byte))
-		result, err := gojsonschema.Validate(scanDoneSchemaLoader, documentLoader)
+		result, validationErr := gojsonschema.Validate(scanDoneSchemaLoader, documentLoader)
 
-		if err != nil {
-			logger.Printf("Error validating input: %v\n", err)
-			break
+		if validationErr != nil {
+			err := fmt.Errorf("error validating input at index %d: %w", i, validationErr)
+			logger.Printf("%v\n", err)
```

Comment:
Suggestion: you don't have to log the errors you return, since the engine will do this for you anyways. So not logging yourself might keep the logs cleaner at the end.